lametro/templates/partials/council_member_table.html

```diff
@@ -15,34 +15,35 @@
+                <td>
+                    <div class="thumbnail-square">
+                        <img src='{{ membership.person.headshot_url }}' alt='{{ membership.name }}' title='{{ membership.name }}' class='img-responsive img-thumbnail' />
+                    </div>
+                </td>
 
-                    <tr id="polygon-{{ membership.post.label | slugify }}" data="{{ membership.name | slugify}}">
+                <td data-order='{{ membership.person.family_name}}'>
+                    {{ membership.person.id | call_link_html | safe }}
+                </td>
 
-                        <td>
-                            <div class="thumbnail-square">
-                                <img src='{{ membership.person.headshot_url }}' alt='{{ membership.name }}' title='{{ membership.name }}' class='img-responsive img-thumbnail' />
-                            </div>
-                        </td>
-                        <td data-order='{{ membership.person.family_name}}'>
-                            {{ membership.person.id | call_link_html | safe }}
-                        </td>
-                        <td data-order='{{ membership.post.label }}'>
-			  {% firstof membership.person.board_office.role membership.role %}
-			  <br>
-                            <span class="small">
-                                 {{ membership.post.acting_label | comma_to_line_break | safe }}
-                            </span><br>
-                            {% if "District" in membership.post.label and "Caltrans District" not in membership.label %}
-                                <span class="badge district-badge">District appointee</span>
-                            {% elif "sector" in membership.post.label  %}
-                                <span class="badge sector-badge">Sector appointee</span>
-                            {% elif "Mayor" in membership.post.label %}
-                                <span class="badge city-badge">Los Angeles</span>
-                            {% else %}
-                            {% endif %}
-                    </tr>
+                <td data-order='{{ membership.post.label }}'>
+                    {% firstof membership.person.board_office.role membership.role %}
+                    <br/ >
 
+                    <span class="small">
+                        {{ membership.post.acting_label | comma_to_line_break | safe }}
+                    </span><br />
+
+                    {% if "District" in membership.post.label and "Caltrans District" not in membership.label %}
+                        <span class="badge district-badge">District appointee</span>
+                    {% elif "sector" in membership.post.label  %}
+                        <span class="badge sector-badge">Sector appointee</span>
+                    {% elif "Mayor" in membership.post.label %}
+                        <span class="badge city-badge">Los Angeles</span>
+                    {% endif %}
```

Comment:
Removing the no-op `{% else %}` here is the only material change. Everything else is formatting.